src/styles/cards.css

```diff
@@ -0,0 +1,82 @@
+@import url("./elements/stars.css");
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  margin: 0 auto;
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name_product {
```

Comment:
```suggestion
.card__name-product {
```

Reply:
fix it everywhere

Reply:
not fixed
<img width="538" alt="Screenshot 2022-12-01 at 12 46 56" src="https://user-images.githubusercontent.com/94289038/205033389-13940ef4-eb9c-490a-9818-e58d2d316d50.png">